src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const ManOrWomen =
+    sex === 'm' ? `${partnerName} is my wife` : `${partnerName} is my husband`;
+
+  const marriedOrNot = isMarried ? ManOrWomen : `I am not married`;
```

Comment:
Better to change the name of this variable to something simpler.
Example: ``marriedOrNot`` - ``partnerStatus``